test/UnitTest/Microsoft.Test.AspNet.OData/Formatter/InheritanceTests.cs

```diff
@@ -8,11 +8,21 @@
 using System.Net.Http;
 using System.Net.Http.Headers;
 using System.Threading.Tasks;
+#if NETFX
```

Comment:
It' easier to manage if you make one #if NETCORE block and one under #else. With them missed like this, sorting & removing no longer works.

Reply:
Fixed. Thanks!